generators/client/templates/angular/src/main/webapp/app/account/password/_password.component.html

```diff
@@ -35,25 +35,37 @@ <h2 jhiTranslate="password.title" translateValues="{username: '{{account.login}}
             <form name="form" role="form" (ngSubmit)="changePassword()" #passwordForm="ngForm">
 
                 <div class="form-group">
-                    <label class="form-control-label" for="password" jhiTranslate="global.form.newpassword">New password</label>
-                    <input type="password" class="form-control" id="password" name="password" #passwordInput="ngModel"
+                    <label class="form-control-label" for="currentPassword" jhiTranslate="global.form.currentpassword">New password</label>
+                    <input type="password" class="form-control" id="currentPassword" name="currentPassword" #currentPasswordInput="ngModel"
+                           placeholder="{{'global.form.currentpassword.placeholder' | translate}}"
+                           [(ngModel)]="currentPassword" minlength=4 maxlength=50 required>
```

Comment:
Since there are no validation message you could remive the min max validation directive as well